docs/readme.md

```diff
@@ -44,7 +44,8 @@ A station is a larger building or area that can be identified by a name. It is u
 		longitude: 52.5250839, // required
 		latitude: 13.3672133 // required
 	},
-	address: 'Europaplatz 1, 10557 Berlin'
+	address: 'Europaplatz 1, 10557 Berlin',
+	regions: ['1234', '2345'] // region ids or region objects, see the section on 'region's
```

Comment:
For additional clarification, we may want to add an "optional" here, just to make sure people don't perceive it as being required.